ndcube/ndcube_sequence.py

```diff
@@ -239,6 +245,91 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def _get_sequence_axes_wcs(self):
+        """
+        Returns a WCS object corresponding to the sequence axis of this NDCubeSequence.
+        It can contain multiple pixel axes that map to the same world axis.
```

Comment:
Isn't this backwards? We have one pixel axis (the sequence axis) but can have many world coordinates (the sequence axis)

Reply:
Right, yes (I'm gonna go check if there are more such instances!)